No module named 'more_itertools'

```
Traceback (most recent call last):
  File "/home/runner/work/MONAI/MONAI/setup.py", line 21, in <module>
    from setuptools import find_packages, setup
  File "/opt/hostedtoolcache/Python/3.9.24/x64/lib/python3.9/site-packages/setuptools/__init__.py", line 18, in <module>
    from setuptools.dist import Distribution
  File "/opt/hostedtoolcache/Python/3.9.24/x64/lib/python3.9/site-packages/setuptools/dist.py", line 46, in <module>
    from . import _reqs
  File "/opt/hostedtoolcache/Python/3.9.24/x64/lib/python3.9/site-packages/setuptools/_reqs.py", line 1, in <module>
    import setuptools.extern.jaraco.text as text
  File "/opt/hostedtoolcache/Python/3.9.24/x64/lib/python3.9/site-packages/setuptools/_vendor/jaraco/text/__init__.py", line 11, in <module>
    from setuptools.extern.jaraco.functools import compose, method_cache
  File "/opt/hostedtoolcache/Python/3.9.24/x64/lib/python3.9/site-packages/setuptools/_vendor/jaraco/functools/__init__.py", line 10, in <module>
    import more_itertools
ModuleNotFoundError: No module named 'more_itertools'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/pythonapp-min.yml
+++ b/.github/workflows/pythonapp-min.yml
@@ -136,6 +136,7 @@ jobs:
       run: |
         which python
         python -m pip install --user --upgrade pip setuptools wheel
+        python -m pip install --user more-itertools>=8.0
     - name: cache weekly timestamp
       id: pip-cache
       run: |

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -2,6 +2,7 @@
 requires = [
   "wheel",
   "setuptools",
+  "more-itertools>=8.0",
   "torch>=2.4.1",
   "ninja",
   "packaging"

